### PR TITLE
Task #185666 chore: Add email service provider support in tjnotification

### PR DESCRIPTION
**Pull request by tushar33**

### CODE REVIEW

File: src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini [hunk 1]
```diff
@@ -286,3 +286,8 @@ COM_TJNOTIFICATIONS_NOTIFICATION_SMS_REMAINING_EXCEEDED="characters exceeded"
 ; Notification - SMS provider template ID
 COM_TJNOTIFICATIONS_FIELD_SMS_TEMPLATE_ID_LABEL="SMS template ID"
 COM_TJNOTIFICATIONS_FIELD_SMS_TEMPLATE_ID_DESC="Set template ID which is proided by SMS provider"
+
+;Notification - Email service provider config
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_PROVIDER="Choose email service provider plugin"
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_PROVIDER_DESC="Choose email service provider plugin to be used to send an emails"
```
Comment (ankush-maherwal) [inline src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini:292]: @tushar33 Choose an Email service provider plugin to be used to send Emails

File: src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini [hunk 2]
```diff
@@ -286,3 +286,8 @@ COM_TJNOTIFICATIONS_NOTIFICATION_SMS_REMAINING_EXCEEDED="characters exceeded"
...
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_DEFAULT_OPTION="Joomla SMTP"
```
Comment (ankush-maherwal) [inline src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini:293]: @tushar33 Joomla SMTP => Joomla Mail
as SMTP is one of the Mail settings in Joomla

File: src/com_tjnotifications/admin/config.xml [hunk 3]
```diff
@@ -6,6 +6,10 @@
 			<option value="1">JYES</option>
 		</field>
 
+		<field name="tjmail_plugin" type="sql" default="" label="COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_PROVIDER" description="COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_EMAIL_PROVIDER_DESC" query="SELECT element, name FROM #__extensions WHERE type='plugin' AND folder='tjmail' AND enabled=1" key_field="element" value_field="name">
```
Comment (ankush-maherwal) [inline src/com_tjnotifications/admin/config.xml:9]: @tushar33 Is the code to use the selected plugin in a different MR?

Comment (tushar33) [reply src/com_tjnotifications/admin/config.xml:9]: That code is in tjnotification library
https://git.tekdi.net/techjoomla/lib_techjoomla/-/merge_requests/154

File: src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini (hunk 1 above)
Comment (tushar33) [reply src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini:292]: done

File: src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini (hunk 2 above)
Comment (tushar33) [reply src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini:293]: done

**Pull request closed without merging**